src/components/RuntimeManager.js

```diff
@@ -341,7 +342,7 @@ export default class RuntimeManager extends PureComponent {
             div({ style: { marginLeft: '0.5rem', paddingRight: '0.5rem', color: colors.dark() } }, [
               div({ style: { fontSize: 12, fontWeight: 'bold' } }, 'Cloud Environment'),
               div({ style: { fontSize: 10 } }, [
-                span({ style: { textTransform: 'uppercase', fontWeight: 500 } },
+                span({ style: { textTransform: 'capitalize', fontWeight: 500 } },
```

Comment:
Things look inconsistent for me now:
![image](https://user-images.githubusercontent.com/5368863/112678702-a22feb80-8e41-11eb-8206-f2cf85b3a3c0.png)

I think `uppercase` is fine..?

Reply:
+1 on the inconsistency. Fixed as in below:

![image](https://user-images.githubusercontent.com/5438223/113054744-c78d6400-9177-11eb-9c5f-c726dc1772bd.png)

Uppercasing signifies strong emphasis to me, at least subconsciously, which I doubt is what we want in this case.

Reply:
Thanks.. am ok with `capitalize` instead of `uppercase`, as long as they're consistent. :)

Reply:
I'm all for consistency whenever possible 😉